tests/test_property.py

```diff
@@ -31,6 +31,24 @@ def test_property(tmp_path):
                                 property_kind = 'net to gross ratio',
                                 indexable_element = 'cells',
                                 uom = 'm3/m3')
+   a3 = np.random.random((grid.nk + 1, grid.nj + 1, grid.ni + 1))
+   p3 = rqp.Property.from_array(model,
+                                a3,
+                                source_info = 'random',
+                                keyword = 'jiggle shared nodes',
+                                support_uuid = grid.uuid,
+                                property_kind = 'length',
+                                indexable_element = 'nodes',
+                                uom = 'm3')
+   a4 = np.random.random((grid.nk, grid.nj, grid.ni, 2, 2, 2))
+   p3 = rqp.Property.from_array(model,
```

Comment:
Should this be p4?

Reply:
Well spotted @emmanesbit !  I've pushed a correction.  (As it happens, that variable is not used afterwards in the test, so no change to behaviour.)